romancal/resample/tests/test_resample.py

```diff
@@ -476,13 +507,30 @@ def test_var_flat_presence(exposure_1, include_var_flat):
         assert hasattr(output_model, "var_flat")
 
 
+def _convert2fitswcs(wcs):
+    forward = wcs.forward_transform
+    crpix = forward[:2].parameters
+    pc = forward[2].matrix.value
+    cdelt = forward[3:5].parameters
+    crval = forward[-1].parameters[:2]
+    fwcs = FITSImagingWCSTransform(
+        projection=models.Pix2Sky_TAN(), crpix=crpix, crval=crval, cdelt=cdelt, pc=pc
+    )
+    custom_wcs = WCS([Step(wcs.input_frame, fwcs), Step(wcs.output_frame, None)])
+    custom_wcs.pixel_shape = wcs.pixel_shape
+    custom_wcs.bounding_box = wcs.bounding_box.bounding_box()
+    return custom_wcs
```

Comment:
What's the goal of this new function?  This seems to only be used in the test code; is the point only to make sure that FITSImagingWCSTransform works?  Would this be better in the create_wcs_object_without_distortion(...) test code?

No strong feelings, just trying to understand.

Reply:
This was mostly to preserve `create_wcs_object_without_distortion` as is. I can change it to create a FITSWCS transform if yo think it's better.